assets/rnadeseq_report.Rmd

```diff
@@ -1688,48 +1870,79 @@ write.table(DE_genes_final_table, "differential_gene_expression/final_gene_table
 }
 ```
 
+
+```{r, print_contrasts, eval=T, results='asis', warning=FALSE, message=FALSE}
+```
+
+
+The first contrast listed should be the experimental condition to test for and the contrast listed at the end should be used as the baseline (control) expression.
+
 ```{r load_config, bootstrap.show.message=FALSE, echo=FALSE, message=FALSE, warning=FALSE}
 logFC_text <- as.character(params$logFC_threshold)
 pval_text <- as.character(params$adj_pval_threshold)
 ```
 
-## List of differentially expressed genes
+### List of DE genes
 
-The analysis of the differential gene expression was performed using `DESeq2`.
+The [secondary bioinformatics pipeline](#secondary-pipeline) uses `DESeq2` to analyze the differential gene expression.
 Genes were considered differentially expressed (DE) when the adjusted p-value was equal to or lower than `r pval_text` (padj <= `r pval_text`).
 Genes were further considered differentially expressed (DE) if the log2 Fold Change in expression between the two considered groups was equal to or above the threshold of `r logFC_text` in absolute terms.
 The adjusted p-value is calculated in the `DESeq2` package with the Benjamini-Hochberg method. P-value adjustment
-helps reduce the number of false postives (not real differentially expressed genes).
+helps to reduce the number of false postives (not real differentially expressed genes).
+
+The following linear model (design) was used in DESeq2 to model gene expression:
+```{r print_model, results='asis'}
+
+lines <- (readLines(params$path_design))
+cat(paste0("`", gsub('condition_', '', lines), "`"), sep = "\n")
 
-The linear model employed to model gene expression in DESeq2 was:
-```{r print_model}
-cat(paste(readLines(params$path_design), collapse=" "))
 ```
 
-The differentially expressed genes are marked with "DE" in the outcome column if they were found
-differentially expressed for any of the contrasts considered.
+
+Uncollapse the table below to show the meaning of different design strategies with example conditions:
+
+<details>
+<summary>Design strategies</summary>
+
+| Design                           | Interpretation                                                                   |
+| -------------------------------- | -------------------------------------------------------------------------------- |
+| `~ treatment`                    | Model effect of treatment only                                                   |
+| `~ treatment + genotype`         | Model additive effects of treatment and genotype (assumes independence of both ) |
+| `~ batch + treatment`            | Control for batch effects when testing treatment                                 |
+| `~ genotype * treatment`         | Include interaction — test if treatment effects differ between genotypes         |
+| `~ batch + genotype * treatment` | Full model, adjusting for batch and testing interaction effects                  |
```

Comment:
```suggestion
| `~ batch + genotype * treatment` | Adjusting for batch and testing interaction effects                  |
```
"Full model" implies the other models are incomplete, thats not optimal I think.